pkg/plugin/predicates/predictor.go

```diff
@@ -201,6 +203,16 @@ func (p *Predictor) init() {
 			return predicates.NewVolumeBindingPredicate(args.VolumeBinder)
 		},
 	)
+
+	// GeneralPredicates are the predicates that are enforced by all Kubernetes components
+	// (e.g. kubelet and all schedulers)
+	p.RegisterFitPredicate(predicates.GeneralPred, predicates.GeneralPredicates)
```

Comment:
I did a cleanup on the predicates listing and kept it consistent with https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithmprovider/defaults/register_predicates.go. The issue was we were using RegisterFit instead of RegisterMandatory earlier for `PodToleratesNodeTaintsPred` and `CheckNodeUnschedulablePred`, now I should get that fixed.